content/blog/rust-gamedev-meetup-33/index.md

```diff
@@ -0,0 +1,65 @@
++++
+title = "Rust Gamedev Meetup 33"
+date = 2023-12-09
+transparent = true
+aliases = ["posts/gamedev-meetup-33"]
++++
+
+<!-- markdownlint-disable single-title heading-increment -->
+<!-- markdownlint-disable no-blanks-blockquote no-emphasis-as-header -->
+<!-- markdownlint-configure-file {"line-length": {"heading_line_length": 120}} -->
+
+![Rust Gamedev Meetup](gamedev-meetup.png)
+
+The 33rd Rust Gamedev Meetup took place on December 9th. You can watch the
+recording of the meetup [here on YouTube][meetup-video]. The meetup took
+place on the Rust Gamedev [YouTube][youtube-stream] and [Twitch][twitch-stream].
+The meetups take place on the second Saturday of every month via the [Rust
+Gamedev Discord server][rust-gamedev-discord] and are also [streamed on
+Twitch][rust-gamedev-twitch].
+
+[rust-gamedev-discord]: https://discord.gg/yNtPTb2
+[rust-gamedev-twitch]: https://twitch.tv/rustgamedev
+[meetup-video]: https://www.youtube.com/watch?v=eItRSKJGV1I
+[youtube-stream]: https://www.youtube.com/@RustGameDevelopment
+[twitch-stream]: https://www.twitch.tv/RustGameDev
+
+## Talks
+
+### [Blue Engine][blue-engine-video] | [@ElhamAryanpur]
+
+<!-- ![Blue engine](blue-engine.png) -->
+
+> Updates about community work for the Blue Engine, as well as future engine
+> features coming up.
+
+**Links:**
+
+- [Blue Engine GitHub repository]
+- [Blue Engine Discord server]
+
+[@ElhamAryanpur]: https://github.com/ElhamAryanpur
+[Blue Engine GitHub repository]: https://github.com/AryanpurTech/BlueEngine
+[Blue Engine Discord server]: https://discord.gg/s7xsj9q
+
+### [Galaxion Trade Wars][galaxion-trade-wars-video] | [@Syn-Nine]
+
+> Space Trading Game made in Rust based on the Trade Wars 2002 BBS game.
+
+**Links:**
+
+- [Galaxion Trade Wars GitHub repository]
+- [Syn-Nine Twitter]
+- [Syn-Nine Website]
+
+[@Syn-Nine]: https://github.com/Syn-Nine
+[Galaxion Trade Wars GitHub repository]: https://github.com/Syn-Nine/galaxion-trade-empire/
+[Syn-Nine Twitter]: https://twitter.com/Syn9Dev
+[Syn-Nine Website]: https://syn9.thehideoutgames.com/
+
+### [Bones Engine] | [@zicklag]
+
+> A 'meta-engine' framework made to facilitate the development of moddable,
+> multiplayer 2D games.
+
+[@zicklag]: https://github.com/zicklag
```

Comment:
```suggestion
**Links:**
- [@zicklag]: https://github.com/zicklag
- [Bone Engine GitHub repository]: https://github.com/fishfolk/bones
- [Jumpy GitHub repository]: https://github.com/fishfolk/jumpy
- [Jumpy's Migration to the Bones Framework]: https://fishfolk.org/blog/jumpy-migration-to-bones-framework/
- [Introducing Lua Scripting in Jumpy]: https://fishfolk.org/blog/introducing-lua-scripting-in-jumpy/
```

Reply:
Nit: Can we put the links below the section, and link to them? For example, like the section above:

![image](https://github.com/rust-gamedev/rust-gamedev.github.io/assets/14791619/a6d2a45a-5b29-4e31-8ade-3a8aca333203)

Reply:
done